src/components/Todo.tsx

```diff
@@ -40,18 +55,39 @@ function Todo(props: { fetchList: any; addTodo: any }) {
     event.preventDefault();
     try {
       const response = await props.addTodo(todo);
-      setNewdata(response["task"]);
+      setNewdata(response);
       setWorking(true);
     } catch {
       setWorking(false);
     }
     setTodo("");
   };
 
+  const [del, setDel] = useState(false);
+  const [id, setId] = useState(0);
+  const deleteItem = async (event: any) => {
+    const buttonId = event.currentTarget.getAttribute("data-testid");
+    setId(buttonId);
+    const canDelete = await deleteTodo(buttonId);
+    setDel(canDelete);
+    // if (del) {
```

Comment:
Going to want to delete the commented code.

Reply:
Sounds good.